sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationClient.cs

```diff
@@ -207,32 +230,31 @@ public virtual async Task<Response<ConfigurationSetting>> SetConfigurationSettin
         public virtual Response<ConfigurationSetting> SetConfigurationSetting(string key, string value, string label = default, CancellationToken cancellationToken = default)
         {
             Argument.AssertNotNullOrEmpty(key, nameof(key));
-            return SetConfigurationSetting(new ConfigurationSetting(key, value, label), default(MatchConditions), cancellationToken);
-        }
 
-        /// <summary>
-        /// Creates a <see cref="ConfigurationSetting"/> if it doesn't exist or overwrites the existing setting in the configuration store.
-        /// </summary>
-        /// <param name="setting">The <see cref="ConfigurationSetting"/> to create.</param>
-        /// <param name="onlyIfUnchanged">If set to true and the configuration setting exists in the configuration store, overwrite the setting
-        /// if the passed-in <see cref="ConfigurationSetting"/> is the same version as the one in the configuration store.  The setting versions
-        /// are the same if their ETag fields match.  If the two settings are different versions, this method will throw an exception to indicate
-        /// that the setting in the configuration store was modified since it was last obtained by the client.</param>
-        /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
-        /// <returns>A response containing the <see cref="ConfigurationSetting"/> written to the configuration store.</returns>
-        public virtual async Task<Response<ConfigurationSetting>> SetConfigurationSettingAsync(ConfigurationSetting setting, bool onlyIfUnchanged = false, CancellationToken cancellationToken = default)
-        {
-            if (setting == null)
-                throw new ArgumentNullException($"{nameof(setting)}");
+            using DiagnosticScope scope = _clientDiagnostics.CreateScope("Azure.Data.AppConfiguration.ConfigurationClient.SetConfigurationSetting");
```

Comment:
Similar to above, this should call through to the SetConfigurationSetting(ConfigurationSetting) overload.

Reply:
Done